modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/transcripts/TranscriptViewModel.kt

```diff
@@ -192,7 +192,9 @@ class TranscriptViewModel @Inject constructor(
             val previousItemEndIndex = if (index == 0) 0 else items[index - 1].endIndex
             val currentItemStartIndex = formattedText.indexOf(currentItem, previousItemEndIndex)
             val isSpeaker = currentItemStartIndex in speakerIndices
-            items.add(DisplayItem(currentItem, isSpeaker, currentItemStartIndex, currentItemStartIndex + currentItem.length))
+            // As we are removing new lines, add a space after each item so copying the text doesn't merge items together.
+            val text = "$currentItem "
+            items.add(DisplayItem(text, isSpeaker, currentItemStartIndex, currentItemStartIndex + currentItem.length))
```

Comment:
This isn't related to the upgrade, but I noticed this while testing when copying text from the transcript, if it goes over a new line, the text is merged. 
For example:
```
Google AI Studio.

It's been awesome
```
would output
```
Google AI Studio.It's been awesome
```
I couldn't find a way to add the new lines back in so this adds an extra space to the line so when it's copied there is a space between them. 
```
Google AI Studio. It's been awesome
```